draft-ietf-quic-qpack.md

```diff
@@ -206,11 +207,30 @@ d = count of entries dropped
 ~~~~~
 {: title="Example Dynamic Table Indexing - Request Stream"}
 
-Entries with an absolute index greater than a frame's Base Index cannot be
-referenced by that frame.  If the decoder encounters a reference to an entry
-which has already been dropped from the table, this MUST be treated as a stream
-error of type `HTTP_QPACK_DECOMPRESSION_FAILED`.  If this reference occurs on
-the control stream, this MUST be treated as a connection error.
+Entries with an absolute index greater than a frame's Base Index can be
+referenced using specific Post-Base instructions.  The relative indices of
+Post-Base references count in the opposite direction from Base Index.
```

Comment:
Is 'count up from Base index' more clear than 'opposite direction'?  It's opposite of the direction of pre-base relative indexes but that may not be obvious from the text.